lib/components/field-key/list.vue

```diff
@@ -186,10 +186,6 @@ export default {
 #field-key-list-table {
   > tbody > tr > td {
     vertical-align: middle;
-
-    .dropdown {
```

Comment:
which dropdowns are these?

Reply:
These are the actions dropdowns in the rows of the web user table and app user table.

Reply:
so what does this change do? they were locked to some width before and now..?

Reply:
oh i see. okay we can fix it later if we care.